packages/plugin-assets/src/commands/assets/init.js

```diff
@@ -1,19 +1,23 @@
 const { flags } = require('@oclif/command');
 const { TwilioClientCommand } = require('@twilio/cli-core').baseCommands;
 const { getPluginConfig } = require('../../pluginConfig');
+const generateProjectName = require('project-name-generator');
+
 const { init } = require('../../init');
 
 class InitCommand extends TwilioClientCommand {
   async run() {
     await super.run();
     try {
+      const { flags } = this.parse(InitCommand);
```

Comment:
Why do you need this here but then down on line 22 you use `this.flags` as opposed to `flags`?

Reply:
Mixed examples in other places I guess! Have updated to always use `this` and not parse for a second time.